sdk/testing/src/main/java/io/opentelemetry/sdk/testing/context/SettableContextStorageProvider.java

```diff
@@ -41,6 +42,7 @@ public Scope attach(Context toAttach) {
     }
 
     @Override
+    @Nullable
```

Comment:
Shouldn't this result in an API change being listed by the API compare task?

Reply:
Believe I ran the build - we've seen the diffs not detect all changes in annotations.